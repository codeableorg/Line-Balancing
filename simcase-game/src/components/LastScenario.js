import React from "react";

function LastScenario({ confirmSubmit, closeSubmit }) {
  return (
    <div>
      <p>You are going to submit your answer. Are you sure?</p>
      <button onClick={confirmSubmit}>Finish and submit</button>
      <button onClick={closeSubmit}>No, I want to continue</button>
    </div>
  );
}

export default LastScenario;
