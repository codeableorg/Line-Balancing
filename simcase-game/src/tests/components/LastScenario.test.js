import React from "react";
import { render } from "@testing-library/react";

import LastScenario from "../../components/LastScenario";

test("LastScenario based on game stage", () => {
  const { asFragment } = render(<LastScenario />);
  expect(asFragment()).toMatchSnapshot();
});
