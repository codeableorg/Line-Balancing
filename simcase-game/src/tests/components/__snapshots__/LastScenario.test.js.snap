// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LastScenario based on game stage 1`] = `
<DocumentFragment>
  <p>
    You are going to submit your answer. Are you sure?
  </p>
  <button
    class="css-rlxcdi-LastScenario"
  >
    Finish and submit
  </button>
  <button
    class="css-rlxcdi-LastScenario"
  >
    No, I want to continue
  </button>
</DocumentFragment>
`;
